README.md

```diff
@@ -33,7 +34,7 @@ This repository includes a browser and a Node.js demo application, and also a st
 You can install and use this library in browsers and nodejs.
 
 ```bash
-npm i @atala/prism-wallet-sdk --save
+npm i @hyperledger/identus-edge-agent-sdk --save
```

Comment:
:fire: TODO
looks like this is what you will be publishing this as in the `package.json` (hence my confidence in changing this here)

Reply:
I've just released @hyperledger/identus-edge-agent-sdk, so we can just apply this change, feel free!